README.md

```diff
@@ -1,6 +1,8 @@
 # React Native Tools
 
 [![Build Status](https://dev.azure.com/vscode-webdiag-extensions/VS%20Code%20WebDiag%20extensions/_apis/build/status/%5BUnit%20tests%5D%20vscode-react-native%20%5Bmaster%5D?branchName=master)](https://dev.azure.com/vscode-webdiag-extensions/VS%20Code%20WebDiag%20extensions/_build/latest?definitionId=60&branchName=master)
+<img src="https://vsmarketplacebadge.apphb.com/version-short/msjsdiag.vscode-react-native.svg">
```

Comment:
Shouldn't this follow the markdown format?

```suggestion
![VS Marketplace version](https://vsmarketplacebadge.apphb.com/version-short/msjsdiag.vscode-react-native.svg)
```

Reply:
Updated this, thanks!